tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc

```diff
@@ -492,6 +493,37 @@ void WarnIfBadDriverJITVersion() {
   });
 }
 
+// Try to load ptx from files defined in the FLAGS. If successful, return true.
+bool MaybeLoadPtxFromFile(const HloModule* module, std::string* ptx) {
+  // If the xla_gpu_ptx_file options is set, be explicit when a file is used
+  // and warn when a file is not used to ease catching typo in filename.
+  std::string prefix = xla::FilenameFor(*module, *ptx);
```

Comment:
I feel it is slightly better to rename them by `s/ptx_filename/matched_filename` and `s/prefix/filename_prefix`.

Up to you though.